### ISSUE TITLE
Make automatic side car injection work properly in multicluster

### ISSUE THREAD

**Issue by sdake** [istio/istio#6092]
automatic side car injection does not function properly in multicluster.  A full mailing list thread with two proposed solutions is covered here: https://groups.google.com/forum/#!topic/istio-environments/7mpzajbGh4c

**Comment by sdake** [comment]
Proposals courtesy of @ayj:
1. replicate the sidecar injector deployment, service, mutatingwebhookconfiguration, configmap, etc. into the remote cluster. 
1. copy the mutatingwebhookconfiguration into the remote cluster and modify that webhookclientconfig to use the explicit URL field instead of the ServiceReference. You'll need to expose the sidecar injector in the local cluster (e.g. via ILB or type=LoadBalancer).

**Comment by sdake** [comment]
@Rigdon had expressed interest in working on the implementation of this work.

**Comment by Rigdon** [comment]
Yup, I'm working on an implementation of (1) and we'll see how it goes.

**Comment by costinm** [comment]
It seems I can't assign either - but thanks for working on it, looking forward for the PR. 
"Assigned" is mostly informative, we don't have any formal use. For P0s we want to be sure
we know who's working on them.

**Comment by Rigdon** [comment]
Quick update on this: I have an installation process that seems to be working very well but I'm currently hung up on #6313 that isn't directly related to multi-cluster but may involve additional work to support multiple clusters with different cluster domains. I've also incorporated an ILB and the RBAC guidelines from [this PR](https://github.com/istio/istio.github.io/pull/1484) but will keep those separate form the eventual PR from this.

**Comment by Rigdon** [comment]
@sdake Can we close this now?

**Comment by ozevren** [comment]
Any update on this? What is the left-over work?
